tests/selenium/options_test.py

```diff
@@ -206,8 +220,68 @@ def test_removed_origin_display(self):
         error_message = "Origin should not be displayed after removal"
         self.assertIsNone(origins, error_message)
 
+    def test_reset_data(self):
+        self.load_options_page()
+        self.select_domain_list_tab()
+
+        # make sure the default tracker list includes many trackers
+        error_message = "By default, the tracker list should contain many trackers"
+        self.assertTrue(
```

Comment:
Should we refactor these three asserts into a function? (Do we call them the same way elsewhere in options tests?)